src/services/conversationsService.js

```diff
@@ -106,20 +106,18 @@ class ConversationsService {
     }
   }
 
-  async createPrivateChat(userId, userObject) {
+  async createPrivateChat(userId, userObject, isEncrypted = false) {
     if (!userId) {
       return;
     }
 
-    const requestData = {
-      type: "u",
-      participants: [userId],
-    };
+    const requestData = { type: "u", participants: [userId] };
+    isEncrypted && (requestData["is_encrypted"] = true);
 
     const chat = await api.conversationCreate(requestData);
     userObject && store.dispatch(addUsers([userObject]));
     store.dispatch(insertChat({ ...chat, messagesIds: null }));
-    store.dispatch(setSelectedConversation({ id: chat._id }));
+    isEncrypted && store.dispatch(setSelectedConversation({ id: chat._id }));
```

Comment:
why we select conv only if it's encrypted?

Reply:
because if we select the chat id before user authorization, then all the triggers for downloading chat content (message_list, user_activity_subscribe) will be triggered